Resources/Prototypes/_NF/Shipyard/rocket.yml

```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: rocket
+  name: SV Rocket
+  description: (pun intended)
+  price: 7000
+  category: Small
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/rocket.yml
+
+- type: gameMap
+  id: rocket
+  mapName: 'SV Rocket'
+  mapPath: /Maps/Shuttles/rocket.yml
```

Comment:
Move the ship file and update the line
``/Maps/_NF/Shuttles/rocket.yml``

Reply:
I searched far and wide throughout the repo, I don't believe the shuttle file you want me to move it to exists, And if it does a full file path would be helpful.

Reply:
Yes, its not there since you need to move it there.

Reply:
Ah my bad, I just realized that my branch was out of sync with the master, I'm making the movements now.